### DISCUSSION TITLE
Pagination Overflow Issue on Grid Page: Scrollbar Appears Due to CSS Break in Latest Version

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#7275] [General]
![image](https://github.com/user-attachments/assets/344b8bad-a11e-4fe6-896b-be5c64e4a8e0)

Serenity Version: 8.7.2

**Comment by VictorTomaili** [comment]
https://github.com/serenity-is/Serenity/pull/7277/files
will be published with next release. you can temporary fix this by adding css rules to your project.

**Reply by dfaruque** [reply]
Thank you for the quick fix.
